docs/modules/user-guide/partials/proc_deploying-a-devfile-registry-with-olm.adoc

```diff
@@ -0,0 +1,55 @@
+[id="deploying-a-devfile-registry_{context}"]
+= Deploying a devfile registry with operator lifecycle manager
+
+[role="_abstract"]
+When you have operator lifecycle manager (OLM) installed on your cluster, use the devfile registry operator to install and deploy a devfile registry.
+
+.Procedure
+..Installing the devfile registry operator
+. Install link:https://github.com/operator-framework/operator-lifecycle-manager[OLM].
+. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/operator-sdk/latest/[operator-sdk CLI].
```

Comment:
Are we referring to `operator-sdk` as a noun or as a command name? It exists as both. As a noun, we'd capitalize it as Operator SDK. As a command name, it should be formatted in monospace (`operator-sdk`).